packages/account/src/Components/file-uploader-container/__tests__/file-uploader-container.spec.tsx

```diff
@@ -26,17 +26,18 @@ describe('<FileUploaderContainer />', () => {
         mock_props = {
```

Comment:
can you please add types to mock_props

Reply:
props are there for `mock_props` you can refer to line number 23 in this file